src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -1,22 +1,22 @@
 package seedu.address.storage;
 
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
 import java.util.stream.Collectors;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import seedu.address.commons.exceptions.IllegalValueException;
+
 import seedu.address.model.person.Address;
 import seedu.address.model.person.CurrentPlan;
```

Comment:
Perhaps there is a missing import?
```suggestion
import seedu.address.model.person.Address;
import seedu.address.model.person.ClientId;
import seedu.address.model.person.CurrentPlan;
```